.env.local

```diff
@@ -13,6 +13,8 @@ ACCOUNT_WEB_URL=http://localhost:3001
 FILE_TRANSFER_API_URL=
 FILE_TRANSFER_API_KEY=
 MAX_FILE_SIZE_BYTES=4194304
+SUPPORTED_MIME_TYPES='text/plain,application/msword,application/pdf,image/jpeg,image/png'
```

Comment:
What about *.jpg or *.docx?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

from my understanding .jpg and jpeg are both captured by  image/jpeg

<img width="878" alt="Screenshot 2020-04-08 at 10 00 05" src="https://user-images.githubusercontent.com/17221386/78765482-f5cece00-797f-11ea-9a81-70d6c1e5a8cf.png">

Reply:
In that case please double check that `*.jpg` files can be handled through `image/jpeg` mime type.

Reply:
The screenshot above demonstrates it